dist/src/main/java/io/camunda/application/StandaloneOperate.java

```diff
@@ -37,7 +39,11 @@ public static void main(final String[] args) {
 
     final var standaloneOperateApplication =
         MainSupport.createDefaultApplicationBuilder()
-            .sources(OperateModuleConfiguration.class, WebappsModuleConfiguration.class)
+            .sources(
+                ApplicationModuleConfiguration.class,
+                CommonsModuleConfiguration.class,
```

Comment:
💭 Can these both consolidated as well? 🤔 As we reference them all the time?

Reply:
Yes, I plan to do it separately also when having Tasklist and Identity included. Right now, we have too many fine-grained configuration classes, which can cause issues, because something might have been forgotten, etc. So my plan is once we have the Single Application with Tasklist and Identity, then I want to re-iterate this part and consolidate that.